src/main/java/jenkins/plugins/git/AbstractGitSCMSource.java

```diff
@@ -470,7 +470,7 @@ protected String getCacheEntry() {
     }
 
     protected static File getCacheDir(String cacheEntry) {
-        Jenkins jenkins = Jenkins.getInstance();
+        Jenkins jenkins = Jenkins.getInstanceOrNull();
```

Comment:
Most of these should actually be `getInstance()` with no null check, but good enough for now.